pkg/controller/replication/replication_controller.go

```diff
@@ -63,6 +63,9 @@ func NewReplicationManager(ctx context.Context, podInformer coreinformers.PodInf
 				Recorder:   eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "replication-controller"}),
 			}},
 			eventBroadcaster,
+			replicaset.ReplicaSetControllerFeatures{
+				EnableStatusTerminatingReplicas: false,
```

Comment:
add a line comment about why this isn't here (field doesn't exist on replication controller, purpose of the field was to propagate status to deployments, which use replicasets, not replication controllers)